build-system/common/git.js

```diff
@@ -169,12 +169,30 @@ function gitCommitterEmail() {
 }
 
 /**
- * Returns the timestamp of the latest commit on the local branch.
+ * Returns list of commit SHAs and their cherry-pick status from master.
+ *
+ * @return {!Array<{sha: string, isCherryPick: boolean}>}
+ */
+function gitCherryMaster() {
+  return getStdout('git cherry master')
+    .trim()
+    .split('\n')
+    .map((line) => ({
+      isCherryPick: line.substring(0, 2) == '- ',
```

Comment:
This can be:
```js
isCherryPick: line.startsWith('- '),
```

Reply:
I went this route because of its relationship to the next line, basically I'm saying "split this at character index 2", I believe this increases the readability in this particular case at not significant additional cost in performance :)